packages/core/src/crypto/keyUtils.ts

```diff
@@ -9,6 +9,9 @@ export function isValidSeed(seed: Buffer, keyType: KeyType): boolean {
     [KeyType.Bls12381g1]: 32,
     [KeyType.Bls12381g2]: 32,
     [KeyType.Bls12381g1g2]: 32,
+    [KeyType.P256]: 32,
+    [KeyType.P384]: 48,
+    [KeyType.P521]: 66,
```

Comment:
@andrewwhitehead I'm not 100% on what the seed and private key sizes should be for `P-256`, `P-384` and `P-521`. Do you know if this is correct?

Reply:
I think it's p521, not p512 (so it's 65.125)

Reply:
They have to be rounded up, as a side note.